docs/plat/cavium/atf_abi.txt

```diff
@@ -0,0 +1,139 @@
+CAVIUM
```

Comment:
None of these contributed docs explain how to use the platform firmware. I would expect a separate document that explains as a minimum:

*   What the platform is and where to get more information.

*   How to build the platform firmware, including configuration options and any pre-requisites.

*   How to install the firmware (or links to this).

Reply:
Will add it.